sdk/servicebus/Microsoft.Azure.ServiceBus/src/Management/QueueDescription.cs

```diff
@@ -308,6 +326,9 @@ public bool Equals(QueueDescription otherDescription)
                 && this.RequiresDuplicateDetection.Equals(other.RequiresDuplicateDetection)
                 && this.RequiresSession.Equals(other.RequiresSession)
                 && this.Status.Equals(other.Status)
+                && this.internalSupportOrdering.Equals(other.SupportOrdering)
```

Comment:
Why would this not be:
```cs
this.SupportOrdering.Equals(other.SupportOrdering)
```
Won't this nullref if internalSupportOrdering is null?